testplan/exporters/testing/http/__init__.py

```diff
@@ -18,14 +18,14 @@
 class HTTPExporterConfig(ExporterConfig):
     """
     Configuration object for
-    :py:class:`HTTPExporter <testplan.exporters.testing.json.HTTPExporter>`
+    :py:class:`HTTPExporter <testplan.exporters.testing.http.HTTPExporter>`
     object.
     """
     @classmethod
     def get_options(cls):
         return {
-            ConfigOption('url', default=None): Or(None,
-                Regex(r'^http[s]?://[\w\d_-]+(:\d{2,5})?.+$', flags=re.I))
+            ConfigOption('url', default=None):
```

Comment:
I don't think default should be None. I think url is a required param so I don't think we need a default?